recap-oss-vienna.mdx

```diff
@@ -97,7 +111,7 @@ Other insightful quotes from the discussion were captured by Arun Gupta in a gre
 
 Devrel foundation have tweeted
 
-https://x.com/linuxfoundation/status/1835590347365163237?s=46&t=oLSbUb4p0O_6BtJKfhCVCg
+https://x.com/linuxfoundation/status/1835590347365163237?s=46\&t=oLSbUb4p0O\_6BtJKfhCVCg
```

Comment:
@miukimiu changing URLs probably fixed the preview but made the URLs not working :)

Reply:
Oops! The auto-prettier fixes kicked in! But I've already pushed the corrections [here](https://github.com/xataio/mdx-blog/pull/324/commits/bd0f27158fea610a27d2d961be72b68fe12940a6).

For future reference, it's always better to use the Markdown syntax for links to avoid prettier issues, like this: `[Link text here](https://link-url-here.org)`.